FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -8,22 +8,25 @@
 import SwiftUI
 
 struct HomeView: View {
+
+
+
     // MARK: - Properties
     @EnvironmentObject var quizManager: QuizManager
     @EnvironmentObject var navigationManager: NavigationManager
     @Binding var presentSideMenu: Bool
-    private let levelItem: [LevelItem] = [
+    private let levelItem: [LevelItem] =     [
         LevelItem(image: Asset.Assets.imgJuniorHighSchool.swiftUIImage, title: "中学生レベル", levelCase: .juniorHighSchool),
         LevelItem(image: Asset.Assets.imgHighSchool.swiftUIImage, title: "高校生レベル", levelCase: .highSchool),
         LevelItem(image: Asset.Assets.imgCollege.swiftUIImage, title: "大学生レベル", levelCase: .college),
         LevelItem(image: Asset.Assets.imgBusinessman.swiftUIImage, title: "社会人レベル", levelCase: .businessman),
         LevelItem(image: Asset.Assets.imgExpert.swiftUIImage, title: "専門家レベル", levelCase: .expert),
         LevelItem(image: Asset.Assets.imgMonster.swiftUIImage, title: "人外レベル", levelCase: .monster)
     ]
-    private let gridItems = [GridItem(.flexible()), GridItem(.flexible())]
+         private let gridItems = [GridItem(.flexible()), GridItem(.flexible())]
 
     // MARK: - Body
-    var body: some View {
+    var body: some View       {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/f0556793b45bd971a01f1d6b9fd707367304df54/FlashEnglish/Sources/Views/Home/HomeView.swift#L29"></span>Opening braces should be preceded by a single space and on the same line as the declaration<br />
<code>opening_brace</code> <code>HomeView.swift:29</code></td>
    </tr>
  </tbody>
</table>